storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="persistent-storage-csi-smb-cifs"]
+= SMB CIFS CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: persistent-storage-csi-smb-cifs
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Server Message Block (SMB) protocol Common Internet File System (CIFS) dialect.
+
+:FeatureName: SMB CIFS CSI Driver Operator
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+After installing the SMB CIFS CSI Driver Operator, {product-title} installs the SMB CIFS CSI Operator and the SMB CIFS CSI driver by default in the `openshift-cluster-csi-drivers` namespace. This allows the SMB CIFS CSI Driver Operator to create CSI-provisioned PVs that mount to SMB CIFS assets.
+
+* The _SMB CIFS CSI Driver Operator_, after being installed, does not create a storage class by default to use to create persistent volume claims (PVCs). However, you can manually create the SMB CIFS `StorageClass`. The SMB CIFS CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand.
```

Comment:
We also need some extra declarion for the limiation and known issues for smb csi driver. 

- Fips enable cluster doesn’t support
https://access.redhat.com/solutions/256053
- Http proxy config does not support
- `onDelete: archive` does not suggested, since still have some bugs https://issues.redhat.com/browse/OCPBUGS-33474

Reply:
As for `onDelete: archive`, we'd like to discourage using archive mode because upstream has important fixes which we have not included (for reason) into OCP for now.

Reply:
Also, it would be nice to make "you can manually create the SMB CIFS StorageClass" a hyperlink to a section about StorageClass (as in https://docs.openshift.com/container-platform/4.15/storage/container_storage_interface/persistent-storage-csi-google-cloud-file.html)

Reply:
@Phaow @mpatlasov Are the two support issues listed above likely to be resolved? If so, I'd recommend putting them in the RNs as bugs. However, if they are likely to persist, then I could add them as not supported items in the driver topic. 

I've added the xref.

Reply:
`onDelete: archive` is going to be resolved soon. Not sure about FIPS and http proxy...

Reply:
For FIPS and http proxy, current SMB protocal does not support yet not sure it will support in the future.

Reply:
@Phaow @mpatlasov Thanks! I'll document FIPS & HTTP proxy limitation, and monitor the `oneDelete:archive `situation to see if we need to add that to the RNs.